src/main/java/org/kcctl/util/Search.java

```diff
@@ -0,0 +1,116 @@
+/*
+ *  Copyright 2021 The original authors
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.kcctl.util;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.function.BiConsumer;
+import java.util.function.Function;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static org.kcctl.service.ConfigInfos.ConfigKeyInfo;
+
+public class Search {
+
+    // Generic records... I'm going to hell for this
+    public record SearchTerm<E>(Function<E, String> extractTerm, BiConsumer<E,String>setTerm)
+    {
```

Comment:
Hum, wondering why the formatter doesn't flag this. The opening brace should at the end of the line above, the matches method should be intended.

Reply:
This is actually the result of the autoformatter; I fixed the styling and then reran `./mvnw process-sources` and it reverted to what's in the PR.

I tried to do some digging into what's causing this but haven't been able to find the part of the formatter (or its dependencies) that's responsible.

Reply:
I see; kinda weird. Did you try updating to the latest version (if there is any newer)? Probably not worth spending too much time on it, at least it's formatted consistently like that, so no friction during any future PRs.